src/pages/page-plan/page-plan.tsx

```diff
@@ -1,159 +0,0 @@
-import { useState, type FC } from "react";
-// components
-import { PagePlanProps } from "./page-plan.interface";
-import { useFormContext } from "react-hook-form";
-import { FormTypes } from "../../types/form-types";
-import { InputPlanCard } from "../../components/input-plan-card/input-plan-card";
-//hooks
-import { useDeviceType } from "../../hooks/use-device-type";
-// assets
-import arcade from "../../assets/images/icon-arcade.svg";
-import advanced from "../../assets/images/icon-advanced.svg";
-import pro from "../../assets/images/icon-pro.svg";
-import { ToogleSwitch } from "../../components/toogle-switch/toogle-switch";
-
-export const PagePlan: FC<PagePlanProps> = ({
-  onBack,
-  onContinue,
-  isMonthly,
-  toggleBilling,
-}) => {
-  const deviceType = useDeviceType();
-  // state
-  const [selectedPlan, setSelectedPlan] = useState("");
```

Comment:
You can remove the `selectedPlan` state since the plan data is already saved in the form. You can use `watch` instead.